src/app/(auth)/reset-password/page.tsx

```diff
@@ -55,39 +63,43 @@ export default function ResetPassword() {
 
   return (
     canReset && (
```

Comment:
can we turn this into a ternary statement? if `canReset` is not true, then we should probably add a text here to tell the user

Reply:
@kyrenetam would it be possible to design a screen for this?

Reply:
for now imma put null there